example/over_react_redux/store.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:redux/redux.dart';
```

Comment:
Would it be worthwhile to move this file (or example/context/store.dart) into a position to be shared between context and over_react_redux? It seems unnecessary to duplicate the whole store. I can see the argument for keeping it separate, but if so then all my nits for example/context/store.dart apply here too :trollface:

Reply:
Yeah, they should be separate ill copy the changes over

Reply:
nevermind lol